File: src/main/java/cz/cvut/fel/aic/amodsim/config/Agentpolis.java
package cz.cvut.fel.aic.amodsim.config;

import java.lang.Boolean;
import java.lang.Integer;
import java.lang.String;
import java.util.Map;

public class Agentpolis {
  public Boolean showStackedEntities;

  public String mapNodesFilepath;

  public Boolean showVisio;

  public Integer simulationDurationInMillis;

  public String mapEdgesFilepath;

  public Agentpolis(Map agentpolis) {
    this.showStackedEntities = (Boolean) agentpolis.get("show_stacked_entities");
    this.mapNodesFilepath = (String) agentpolis.get("map_nodes_filepath");
    this.showVisio = (Boolean) agentpolis.get("show_visio");
    this.simulationDurationInMillis = (Integer) agentpolis.get("simulation_duration_in_millis");
    this.mapEdgesFilepath = (String) agentpolis.get("map_edges_filepath");
  }
}

File: src/main/python/amodsim/config/agentpolis.py
<<<<<<< HEAD

class Agentpolis:
    def __init__(self, properties: dict=None):
        self.show_stacked_entities = properties.get("show_stacked_entities")
        self.show_visio = properties.get("show_visio")
        self.simulation_duration_in_millis = properties.get("simulation_duration_in_millis")
        self.map_nodes_filepath = properties.get("map_nodes_filepath")
        self.map_edges_filepath = properties.get("map_edges_filepath")


        pass

=======

class Agentpolis:
    def __init__(self, properties: dict=None):
        self.map_edges_filepath = properties.get("map_edges_filepath")


        pass

>>>>>>> vga_ridesharing2
